src/report_reader.cpp

```diff
@@ -279,6 +279,31 @@ std::vector<NodeID> ReportReader<T>::Population::getNodeIds() const {
     return nodes_ids_;
 }
 
+template <typename T>
+Selection::Values ReportReader<T>::Population::node_ids_from_selection(
+    const nonstd::optional<Selection>& selection) const {
+    // Simplify selection
+    // We should remove duplicates
+    // And when we can work with ranges let's sort them
+    // auto nodes_ids_ = Selection::fromValues(node_ids.flatten().sort());
```

Comment:
I don't understand this comment

Reply:
I wrote it, but i don't remember why.
In my head it means, one interesting improvement could be to group ranges for having a single read